installers/rpm-code/codes.sh

```diff
@@ -27,7 +27,7 @@ codes_cmake_fix_lib_dir() {
     perl -pi -e '/include\(GNUInstallDirs/ && ($_ .= q{
 set(CMAKE_INSTALL_LIBDIR "lib" CACHE PATH "Library installation directory." FORCE)
 GNUInstallDirs_get_absolute_install_dir(CMAKE_INSTALL_FULL_LIBDIR CMAKE_INSTALL_LIBDIR)
-})' CMakeLists.txt
+})' $(find . -name CMakeLists.txt)
```

Comment:
If there are spaces in directories, this will break. It needs to be done with xargs, e.g.

```sh
find ... -print0 | xargs -0 perl -pi ... 
```

I don't think this will break anything. pymesh(?) is the only one with lots of makefiles. 

This is where we need a build test system, because we really should rebuild all codes with this fixup to verify they build correctly. Only codes with more than one CMakeLists.txt matter so that's a check we could add in this script and output a warning if the files actually differ. 

You could change the `-pi` to `.pi.codes_fix`. Then do something like this after the edit:

```sh
find ... -print0 | xargs -0 -n 1 -I % diff -c %.codes_fix %
```

If there is any output, you'll at least see if it is more than one file, and if it is screwed up.